dsref/version_info_test.go

```diff
@@ -0,0 +1,55 @@
+package dsref
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/qri-io/dataset"
+)
+
+func TestConvertToVersionInfo(t *testing.T) {
+	ds := &dataset.Dataset{
+		Peername: "a",
+		Name:     "b",
+		Path:     "c",
+		Commit: &dataset.Commit{
+			Timestamp: time.Time{},
```

Comment:
Would be nicer to use a non-zero value here, to verify that the test is actually copying the field.